### ISSUE TITLE
any example for rule-id enabling ?

### ISSUE THREAD

**Issue by zufukatana** [XiaoliChan/wmiexec-Pro#3]
python3 wmiexec-pro.py administrator:password@192.168.1.1 firewall -rule-id (ID from search port) -rule-op [enable/disable/remove]

tested this with firewall -rule-id Netlogon-NamedPipe-In -rule-op enable. got not params for -rule-op

**Comment by XiaoliChan** [comment]
Sorry, I rename the `-rule-op` to `-aciton`, forget update the readme.

**Comment by XiaoliChan** [comment]
Just update the readme file :)

**Comment by zufukatana** [comment]
yeah digged into source code and found -action , it works, I forgot to tell ya here :) thanks anyways.
